src/humanloop/client.py

```diff
@@ -78,11 +90,124 @@ def __init__(
             follow_redirects=follow_redirects,
             httpx_client=httpx_client,
         )
+
+        self._tracer_provider = TracerProvider(
+            resource=Resource(
+                attributes={
+                    "instrumentor": "humanloop.sdk",
+                }
+            ),
+        )
+        instrument_provider(provider=self._tracer_provider)
+        self._tracer_provider.add_span_processor(
+            HumanloopSpanProcessor(
+                exporter=HumanloopSpanExporter(
+                    client=self,
+                )
+            ),
+        )
+        tracer = self._tracer_provider.get_tracer("humanloop.sdk")
+        set_tracer(tracer)
+
         eval_client = ExtendedEvalsClient(client_wrapper=self._client_wrapper)
         eval_client.client = self
         self.evaluations = eval_client
         self.prompts = ExtendedPromptsClient(client_wrapper=self._client_wrapper)
 
+    def prompt(
+        self,
+        # TODO: Template can be a list of objects
+        path: Optional[str] = None,
```

Comment:
We should be re-using the enums and types from the SDK here instead of redefining literals for things like providers